warehouse/age-off-utils/src/main/java/datawave/age/off/util/AgeOffCsvToMatchPatternFormatterConfiguration.java

```diff
@@ -0,0 +1,159 @@
+package datawave.age.off.util;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.Reader;
+import java.io.Writer;
+import java.nio.file.Files;
+import java.util.Map;
+
+public class AgeOffCsvToMatchPatternFormatterConfiguration {
+    private boolean shouldCloseWriter;
+    private String staticLabel;
+    private char quoteCharacter;
+    private boolean shouldQuoteLiteral;
+    private boolean useColons; // when false, use equals sign
+    private boolean shouldPadEquivalence; // when false, no spaces around equals sign
+    private boolean shouldLowerCaseLiterals;
+    private boolean shouldUpperCaseLiterals;
+    private Map<String,String> valueMapping;
+    private Writer writer;
+    private BufferedReader reader;
```

Comment:
If you are down with the proposal about Confs just holding the raw data and Formatters printing, then I'd suggest we get rid of the reader object in the Conf. Can we just read the CSV data during the building of the conf object and put the data in the conf - the most straightforward would be as Strings for each line.

If you wanted to stream these readers for memory reasons, then we could do something slicker where you just give the Conf an Iterator<String> so it's ignorant about the reader under the covers. Then the iterator could invoke the BufferedReader.readLine() and handle closing the stream at the end.

Reply:
The Iterator idea worked better than I expected.  Thanks for the feedback!